### DISCUSSION TITLE
是否可以 maven 类的提供打包依赖

### DISCUSSION THREAD

**Discussion by A2Data** [astral-sh/rye#372] [Ideas]
通过 yaml 之类的文件来管理依赖，用过 rye 来运行和编译， 只有一套或者本地， 我认为很多时候都是因为整了好多虚拟环境，所以才导致 各种异常问题， 从目前的体验来看，不同的版本 不同的包，包括pip官方 也有很多不维护的， 希望可以有个稳定的工具来解决这个问题，使得项目更加稳定，长期，更加专注到业务逻辑和性能上。

**Comment by ischaojie** [comment]
Please communicate in English. :)

**Reply by A2Data** [reply]
Is it possible to provide packaging tools similar to Maven, and then simply install rye like pyenv, using a set of tools to manage the global environment, including dependency compilation and running? I often suffer from the environment and packages
